### PR TITLE
Updated for Planetary Software

### PR DESCRIPTION
**Pull request by rbeyer**
I removed a bunch of ASC-specific stuff, and fixed the vote-to-close discussion logic.

### CODE REVIEW

File: TSC-Charter.md [hunk 1]
```diff
@@ -84,25 +63,25 @@ including:
 * Conduct guidelines.
 * Maintaining the list of additional Collaborators.
 * Development process and coding standards.
-* Mediating technical conflicts between Collaborators or ASC Organization
+* Mediating technical conflicts between Collaborators or Planetary Software Organization
```
Comment (jlaura) [inline TSC-Charter.md:67]: Do we want to add 'Maintaining a list of affiliate projects'?

File: TSC-Charter.md [hunk 2]
```diff
@@ -84,25 +63,25 @@ including:
...
+The TSC will establish and maintain a development process for the Planetary Software Organization's Projects. The development process will establish guidelines
 for how the developers and community will operate. It will, for example,
 establish appropriate timelines for TSC review (e.g. agenda items must be
 published at least a certain number of hours in advance of a TSC
 meeting).
 
-There will be multiple Projects under the ASC Software Organization organized by
+There will be multiple Projects under the Planetary Software Organization organized by
 modules or subsystems. The TSC is responsible for organizing the Project
 structure, including possibly the creation and alignment of sub-Projects.
```
Comment (jlaura) [inline TSC-Charter.md:84]: Should the language here be more along the lines of 'including possibly supporting the creation and alignment of sub-Projects'. The heritage of this doc coming from Node with such a homogeneous environment comes through here pretty strongly.

**Review by jlaura — APPROVED** [review]

File: TSC-Charter.md (hunk 1 above)
Comment (rbeyer) [reply TSC-Charter.md:67]: Sure.

File: TSC-Charter.md (hunk 2 above)
Comment (rbeyer) [reply TSC-Charter.md:84]: Yes, I was just going through these during while half-paying attention to another meeting and trying to figure out what the minimal changes were to 'flip' the document.  I wasn't parsing deeply.  I think a change like that would be good.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**